samples/Azure.Management.Storage/Generated/BlobContainerContainer.cs

```diff
@@ -221,7 +221,7 @@ Page<BlobContainer> FirstPageFunc(int? pageSizeHint)
                 scope.Start();
                 try
                 {
-                    var response = _restClient.List(Id.ResourceGroupName, Id.Name, maxpagesize, filter, cancellationToken: cancellationToken);
+                    var response = _restClient.List(Id.ResourceGroupName, Id.Name, Convert.ToString(pageSizeHint), filter, cancellationToken: cancellationToken);
```

Comment:
I prefer if we use a directive to change the parameter type in the swagger to be int? instead of adding this convert here.